TheBTeam.BLL/Services/TransactionService.cs

```diff
@@ -9,43 +9,43 @@ namespace TheBTeam.BLL.Services
     public class TransactionService
     {
         private static List<Transaction> Transaction = new List<Transaction>();
-        //private List<Transaction> Transaction = new List<Transaction>
+        //private list<transaction> transaction = new list<transaction>
```

Comment:
i believe pascal case used previously was correct as per naming convention, pascal case for types and so on.

Although its of no consequence, i got the generation and serialization of transactions down its just putting them in the mvc proves to be a bit challenging, mostly because my present code is spaghetti,  and also because im dumb

Reply:
Should be_transactions

Reply:
But still this is commented it's ain't my code